Files/Interacts/Interaction.cs

```diff
@@ -1278,6 +1278,21 @@ private void NewFile(ShellNewEntry itemType)
 
         public void ClearAllItems(BaseLayout contentPage) => contentPage.ClearSelection();
 
+        public async void PinItemToStart_Click(object sender, RoutedEventArgs e)
```

Comment:
Hey @winston-de! Could you avoid adding any more code to the `Interaction` class? The class is in progress of being removed by me and #4016 is a starting point of this.

Reply:
@d2dyno1 sure, sorry about that. Where should I put it instead?

Reply:
@winston-de I guess in the code behind of the GenericFileBrowser and GridViewBrowser. It'll be easier for me to later update the pattern

Reply:
@d2dyno1 alright changes made

Reply:
Thank you!